src/lib_ccaligner/params.cpp

```diff
@@ -611,8 +597,3 @@ void Params::printParams()
     std::cout<<"\n\n=====================================================\n\n";
 
 }
-
```

Comment:
At that time, it was favoured to keep empty destructors for the people using it as API to simply customise it as they want.

Reply:
@saurabhshri I don't understand. So you mean people using `Params` should be able to modify the destructor to fit their needs?

Reply:
@Rakete1111 They should be able to modify anything to fit their needs :)

Reply:
@saurabhshri But then they can also add the destructors themselves, no?

Reply:
@Rakete1111 Yes, sure! :) No problem.